### ISSUE TITLE
Instrument glitches with HDR and FlightGear next

### ISSUE THREAD

**Issue by ranguli** [c172p-team/c172p#1519]
My apologies if these are known issues. I'm running the `next` branch of FlightGear and the latest commit on the `master` branch of the C172P. Using HDR on an AMD graphics card (Radeon 760M). The glitches are not present when HDR is disabled.

Turn indicator and compass:
![image](https://github.com/user-attachments/assets/9ae5ec0e-5f92-4e9e-9a85-f7128e6cecad)

Cubes appearing on fuel tank selector:
![image](https://github.com/user-attachments/assets/da0ff86d-c508-4351-b139-dab262b3e8f8)

Border around carb heat knob:
![image](https://github.com/user-attachments/assets/6050784c-b34e-4785-8876-adc00d51103e)

Right side of the yoke is untextured:
![image](https://github.com/user-attachments/assets/8dfe7832-5f21-40ec-8abc-aeccf7235d4b)

DME screen is greyed out:
![image](https://github.com/user-attachments/assets/c4804043-f2df-4b56-8253-aba6424afe94)

Nav instruments:
![image](https://github.com/user-attachments/assets/b3e61b80-7792-4d6d-887b-8f471633b13f)

Please let me know if there is any additional information I can provide.

Thanks!

**Comment by wlbragg** [comment]
Sorry for the delay in getting back to you. The default version of the c172p is not HDR compatible. But I did make a branch that is a little more compatible, but not perfect see: https://github.com/c172p-team/c172p/tree/PBR
It may be awhile before we get a complete HDR version.

FYI: the carb heat is normal, it is actually designed with that square and that border. It may get changed in the future as I can't find any example of that switch anywhere to support its existence. Everything i found uses a square, but it is a molded plastic. Maybe back in the day they added the square plate to eliminate vibration?

I have been aware that the yoke right side didn't look right in certain lighting. So I decided to check it out a little closer just now, it is a flipped normals issue. I corrected it and I am going to push it shortly. I will push the fix to the main and PBR branches.

Thank you for your report.

**Comment by wlbragg** [comment]
Fix to yoke normals pushed to main and PBR branches. Please do pull the PBR branch and check it out. Let me know how it works for you. It is a start, but certainly is likely to have some issues. Thanks again.

**Comment by callahanp** [comment]
The latest commit on next:
./fgdata:      6dd554a376b151e69eb7ceb7563063c759dc64bb  c172p 2024.2.0 initial from c172p-team

Llighting for radios is very dim, and no lighting at all for instruments.
Zooming in does seem to brighten the radio indication.  Zooming out dims the frequencies on the kx165 so they are barely visible.

Adjusting the radio lighting up and down has no effect, 0 to 100% all the same.  This is similar to other click/drag items, which do not work:  Show/Hide Yoke, 

Setting the time to night results in a completely dark screen, no instruments, no radio frequencies visible.

Also noted that clicking on hide/show yoke, and selecting Cessna C172P -> Hide/Yokes had no effect.

**Comment by wlbragg** [comment]
Thanks Pat,

Consider all lighting completely broken on the c172p on next.

We need to organize issues related to the c172p-next (HDR) because there are going to be many.

1) I don't know GitHub well enough to know if there is a method to distinguish issues by branch. If that is possible then we need to do that. At the least maybe we can just use a tag at the start of the issue headline like HDR - Bla bla bla.

2) Issues need to be kept separate from one another as always. This issue heading is too broad for effectively working the issues. I or someone else needs to separate issues into a more manageable detailed accounting.
For example, for lighting.
a) interior cabin
b) instrument
c) exterior
d) lightmaps (if applicable)
So, yoke no longer hiding would be a single issue unless it can be linked to other "like" issues. In this case it happens to be a transparency issue. It is not a simple select which would still work. @hbeni I don't know if you were the creator of the hide yoke code or not. Maybe you already know its transparency issue with HDR.

3) We need to have a discussion on how to best proceed with getting the 172 organized to work with HDR/PBR. Such as how to utilize gltf. How to organize any PBR texturing, do we need new lightmaps or are they still relevant, what type of lighting effects do we have now. I'm sure the list is long.

For everyone's information in case it isn't obvious, the 2024.2 c172p branch is the official fgfs 2024.2 next aircraft (until or unless changed by the powers to be). This is our starting point for a complete revision to HDR/PBR. I don't think we should just hack it to work in HDR, it needs a complete, well thought out, overhaul.

Comments and suggestions are most welcome.

**Comment by hbeni** [comment]
> I don't know if you were the creator of the hide yoke code or not. Maybe you already know what its transparency issue is with HDR?

Yes, I made a transparency addition to the simple "unselect". It is no black magic, however, and a very simple material animation. I hugely think this needs to be fixed in the renderer itself.
See: https://github.com/c172p-team/c172p/commit/e4fd8e4b1babf75b411e66332da10c69951171b4

**Comment by hbeni** [comment]
Regarding organizing: We might try out the "projects" feature of github: https://docs.github.com/en/issues/tracking-your-work-with-issues/configuring-issues/planning-and-tracking-work-for-your-team-or-project#adding-issues-to-a-projects
